bridge/server/services/data-service.ts

```diff
@@ -327,6 +327,17 @@ export class DataService {
     return Yaml.parse(shipyard);
   }
 
+  public async createSubscription(integrationId: string, subscription: UniformSubscription, webhookConfig: WebhookConfig): Promise<void> {
+    const response = await this.apiService.createSubscription(integrationId, subscription);
+    const subscriptionId = response.data.id;
+    await this.saveWebhookConfig(webhookConfig, subscriptionId);
```

Comment:
Following the path from component to this point webhookConfig is always optional, except here. Because when `req.body.webhookConfig` in `api/index.ts` is undefined it will be undefined here.

Did you also consider that subscriptions are created / updated without a webhook config? Now it seems like it tries to always save a webhook config. Also in `saveWebhookConfig` I could see no mechanism to just not save the config as it is also not optional there.

Reply:
You are right. Forgot to make it optional. Fixed it